tests/manual/test-procedure.rst

```diff
@@ -446,6 +446,25 @@ Test Case: Autocomplete
 - [ ] Disable autocomplete in "Options" and confirm that autocomplete no longer offers suggestions.
 
 
+Test Case A11y: Keyboard focus order follows the visual layout
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+- [ ] Using your keyboard, navigate through the menu and any submenus using the `tab` and `enter` or `space` keys
+and confirm  they follow natural order of the page elements eg.left to right and top top bottom.
+- [ ] Using your keyboard, navigate through the modal windows using the `tab` and `enter` or `space` keys
+and confirm  they follow natural order of the page elements eg.left to right and top top bottom.
+
+
+Test Case A11y: Esc key returns focus to menu
+'''''''''''''''''''''''''''''''''''''''''''''
+- [ ] Type something in the text editor, then confirm the `Esc` key returns the focus to the menu from the text editor.
+- [ ] Enter the Load/Save modal window and confirm the `Esc` key closes the open modal.
+
+
+Test Case A11y: Using a screenreader
+''''''''''''''''''''''''''''''''''''
+- [ ] If you are using a Mac, enable the Voiceover tool https://support.apple.com/en-gb/guide/voiceover-guide/vo2682/web and repeat the A11y tests using Voiceover.
```

Comment:
Are there any other accessibility tools like this in windows? Will this test be a mac-only test at the moment?

Reply:
It looks like https://www.nvaccess.org/download/ is free to use, though I have no experience of using it and I can't test as I think Catalina has broke Virtualbox. Perhaps for the moment it's mac only?

Reply:
Okay, we might have to revisit that in the future, but should be fine now.